cc/private/toolchain/cc_configure.bzl

```diff
@@ -33,12 +33,17 @@ def cc_autoconf_toolchains_impl(repository_ctx):
     should_detect_cpp_toolchain = "BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN" not in env or env["BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN"] != "1"
 
     if should_detect_cpp_toolchain:
+        if repository_ctx.os.name.find("windows") != -1:
```

Comment:
can you make this `repository_ctx.os.name.lower().find(...)`

that would at least be consistent with, e.g. [rules_java](https://github.com/bazelbuild/rules_java/blob/113827798fc6ddcd180ebe365b8b4951eb628def/toolchains/local_java_repository.bzl#L134)